GitUpKit/Utilities/GIAppKit.h

```diff
@@ -36,6 +36,7 @@ extern NSString* const GICommitMessageViewUserDefaultKey_EnableSpellChecking;
 
 @interface NSView (GIAppKit)
 - (void)replaceWithView:(NSView*)view;  // Preserves frame and autoresizing mask
+- (NSImage*)snapshotImage;
```

Comment:
The name is a bit ambiguous: is snapshot a verb here or a noun? If the latter, this should be a property instead.

Maybe `-(NSImage*)takeSnapshot`?